src/mscorlib/corefx/System/Globalization/CalendarData.Unix.cs

```diff
@@ -38,10 +67,53 @@ internal static int GetCalendars(String localeName, bool useUserOverride, Calend
 
         private static bool SystemSupportsTaiwaneseCalendar()
         {
-            // TODO: Implement this.
-            return false;
+            return true;
         }
 
         // PAL Layer ends here
+
+        private static bool GetCalendarInfo(string localeName, CalendarId calendarId, CalenderDataType dataType, out string calendarString)
+        {
+            calendarString = null;
+
+            const int stringSize = 80;
```

Comment:
I think this is a liability.  I don't really like the fact that windows does something similar with a fixed size buffer (especially since the native calls never seem to say anything about there being a limit on the returned data).  Can we update the native code to set errno on insufficient buffers, detect that here and expand our builder and retry?